src/main/kotlin/com/openlattice/chronicle/services/ScheduledTasksManager.kt

```diff
@@ -17,8 +22,18 @@ private const val DEVICES_INFO_REFRESH_INTERVAL = (60 * 1000L) // 1 minute
  * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
-class ScheduledTasksManager {
-    private val logger = LoggerFactory.getLogger(ScheduledTasksManager::class.java)
+class ScheduledTasksManager(
+        private val storageResolver: StorageResolver
+) {
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(ScheduledTasksManager::class.java)
+
+        private val GET_SYSTEM_APPS_SQL = """
+            SELECT * FROM ${SYSTEM_APPS.name}
```

Comment:
It's preferable to list out columns, even if you grab the SYSTEM_APPS.columns.joinToString